pkg/recommender/engine.go

```diff
@@ -95,10 +95,26 @@ func (e *Engine) recommendMaster(provider, service string, req ClusterRecommenda
 	if layoutDesc == nil {
```

Comment:
please negate this condition and "return early" here instead of wrapping the code ...

Reply:
done, thx